lamindb/_feature.py

```diff
@@ -50,25 +74,13 @@ def __init__(self, *args, **kwargs):
         raise ValueError("Please pass dtype!")
```

Comment:
Since you're touching this anyways and you added `FEATURE_DTYPES`, could you use this to fix https://github.com/laminlabs/lamindb/issues/2212 , please?

Reply:
Done!